.changeset/clever-experts-return.md

```diff
@@ -0,0 +1,5 @@
+---
+"@neo4j/cypher-builder": patch
+---
+
+Support for UNWIND statement after CALL { ... }
```

Comment:
No changeset for the library needed?

Reply:
The changeset at #2455 already covers cypher changes, so no sense in adding a second one if we are releasing both top level and field in one go